src/SDKs/EventGrid/management/Management.EventGrid/Properties/AssemblyInfo.cs

```diff
@@ -7,8 +7,8 @@
 [assembly: AssemblyTitle("Microsoft Azure EventGrid Management Library")]
 [assembly: AssemblyDescription("Provides developers with a library to create and manage all Azure EventGrid resources.")]
 
-[assembly: AssemblyVersion("1.2.0.0")]
-[assembly: AssemblyFileVersion("1.2.0.0")]
+[assembly: AssemblyVersion("1.3.0.0")]
```

Comment:
@kalyanaj starting version 2.0, it is advisable you do not bump up AssemblyVersion for every minor/revision version bump. Unless you have specific reason and you want your developers to take a hand dependency on particular AssemblyVersion.
If you want to do this starting next major version bump, mention it over here and this PR is ready to be merged.